private/bufpkg/bufapiclient/bufapiclient.go

```diff
@@ -12,45 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package bufapiclient provides client-side gRPC constructs.
+// Package bufapiclient provides client-side Connect constructs.
```

Comment:
Random thought, maybe we could combine `bufapiclient` and `buftransport` so it's easier for others to discover the mapper functions.

Reply:
How are they used in core?

Reply:
Oops, ignore this comment. `apiclient` is generated so it doesn't make sense.